fpmsyncd/routesync.cpp

```diff
@@ -23,9 +23,6 @@ RouteSync::RouteSync(RedisPipeline *pipeline) :
     m_vnet_tunnelTable(pipeline, APP_VNET_RT_TUNNEL_TABLE_NAME, true),
     m_warmStartHelper(pipeline, &m_routeTable, APP_ROUTE_TABLE_NAME, "bgp", "bgp")
 {
-    m_nl_sock = nl_socket_alloc();
-    nl_connect(m_nl_sock, NETLINK_ROUTE);
-    rtnl_link_alloc_cache(m_nl_sock, AF_UNSPEC, &m_link_cache);
```

Comment:
Could this netlink connection and cache allocation be kept here and reused at RouteSync::getIfName()?  So no need to  create netlink connection and alloc cache for each route message?

Reply:
@jipanyang This is a very good idea!